Content.Server/Preferences/Managers/ServerPreferencesManager.cs

```diff
@@ -41,7 +42,7 @@ public void Init()
             _netManager.RegisterNetMessage<MsgSelectCharacter>(HandleSelectCharacterMessage);
             _netManager.RegisterNetMessage<MsgUpdateCharacter>(HandleUpdateCharacterMessage);
             _netManager.RegisterNetMessage<MsgDeleteCharacter>(HandleDeleteCharacterMessage);
-            _sawmill = _log.GetSawmill("prefs");
+            _sawmill = _logManager.GetSawmill("SawmillName");
```

Comment:
I think this should be a variable `SawmillName` rather than a string `"SawmillName"`

Reply:
yes